x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -1161,7 +1161,6 @@
     "data.advancedSettings.timepicker.refreshIntervalDefaultsText": "時間フィルターのデフォルト更新間隔「値」はミリ秒で指定する必要があります。",
     "data.advancedSettings.timepicker.refreshIntervalDefaultsTitle": "タイムピッカーの更新間隔",
     "data.advancedSettings.timepicker.thisWeek": "今週",
-    "data.advancedSettings.timepicker.timeDefaultsText": "時間フィルターが選択されずにKibanaが起動した際に使用される時間フィルターです",
```

Comment:
There is also `x-pack/plugins/translations/translations/zh-CN.json` that needs to be updated?

Reply:
Thanks! Don't know how I missed that.